JsonSchema/JsonSchema.cs

```diff
@@ -21,7 +106,11 @@ public class JsonSchema : IBaseDocument
 {
 	private const string _unknownKeywordsAnnotationKey = "$unknownKeywords";
 
+#if NET6_0_OR_GREATER
+	private static readonly HashSet<SpecVersion> _definedSpecVersions = [..Enum.GetValues<SpecVersion>()];
+#else
 	private static readonly HashSet<SpecVersion> _definedSpecVersions = [..Enum.GetValues(typeof(SpecVersion)).Cast<SpecVersion>()];
+#endif
```

Comment:
Can we move the enum value fetching into a helper function somewhere?  It would clean this up a bit.  We get enum values elsewhere also.

Reply:
Can do.